tests/ensure_provider_tests.py

```diff
@@ -95,6 +95,11 @@
     },
     "playbooks/tests_reapply.yml": {},
     "playbooks/tests_route_table.yml": {},
+    "playbooks/tests_route_type.yml": {
+        MINIMUM_VERSION: "'1.36.0'",
```

Comment:
What happens when the test runs against an older version of NM? It does not seem that the module protects against this.

Reply:
right, I should add an validation function.